### PR TITLE
Update database!

### PR DESCRIPTION
**Pull request by margocramer**

### CODE REVIEW

File: spec/models/commute_spec.rb [hunk 1]
```diff
@@ -8,15 +8,13 @@
       it { should have_valid(:ending_location).when("Attleboro Station") }
       it { should_not have_valid(:ending_location).when(nil, "")}
 
-      it { should have_valid(:time).when("Early Morning", "Morning", "Noon", "Afternoon", "Evening", "Night") }
+      it { should have_valid(:time).when("Early Morning", "Morning", "Afternoon", "Early Evening", "Evening", "Night") }
       it { should_not have_valid(:time).when(nil, "")}
```
Comment (bkincart) [inline spec/models/commute_spec.rb]: It would also be worth adding a test here for something that is NOT nil or an empty string, but instead a string that doesn't match your selected times

File: spec/models/review_spec.rb [hunk 2]
```diff
@@ -4,18 +4,18 @@
   it { should belong_to :user }
   it { should belong_to :commute }
 
-  it { should have_valid(:tally_count).when(1, 2, 3, 4, 5) }
-  it { should_not have_valid(:tally_count).when("cabbage")}
+  it { should have_valid(:review_stars).when(1, 2, 3, 4, 5) }
+  it { should_not have_valid(:review_stars).when("cabbage")}
```
Comment (bkincart) [inline spec/models/review_spec.rb]: Also worth adding a test here for if review_stars is left blank

File: spec/controllers/api/v1/commutes_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe Api::V1::CommutesController, type: :controller do
+  let!(:first_user) { User.create(first_name: "Alan", last_name: "Stevens", email: "alan@stevens.com", password: "mypassword", username: "sathvik")}
+  let!(:second_user) { User.create(first_name: "Adam", last_name: "Salts", email: "adam@salts.com", password: "password", username: "evilchicken")}
+
+  let!(:first_commute) { Commute.create(starting_location: "East Cambridge", ending_location: "Plymouth", mode: "Automobile", time: "Afternoon", user_id: 1) }
+  let!(:second_commute) { Commute.create(starting_location: "Carver", ending_location: "Woburn", mode: "Walk", time: "Morning", user_id: 2) }
+
+  xdescribe "GET#index" do
```
Comment (bkincart) [inline spec/controllers/api/v1/commutes_controller_spec.rb]: This test looks great - any reason it's being skipped at this time?

File: spec/controllers/api/v1/commutes_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      expect(returned_json[1]["ending_location"]).to eq "Woburn"
+      expect(returned_json[1]["time"]).to eq "Morning"
+      expect(returned_json[1]["rating"]).to eq 2
+    end
+  end
+
+  xdescribe "GET#show" do
+    it "should return reviews for the commute being shown" do
+
+      get :show
```
Comment (bkincart) [inline spec/controllers/api/v1/commutes_controller_spec.rb:33]: Just a note here (I know this is pending for now) - you will want to pass in a particular commute when calling the `#show` action!

**Review by bkincart — CHANGES_REQUESTED** [review]
Just a few edge cases to add to your test, and one test to unmark as pending, and this should be good to go!

**Review by bkincart — APPROVED** [review]
